utils/parse.js

```diff
@@ -81,6 +81,7 @@ exports.default = function parse(path, content, context) {
   // "project" or "projects" in parserOptions. Removing these options means the parser will
   // only parse one file in isolate mode, which is much, much faster.
   // https://github.com/import-js/eslint-plugin-import/issues/1408#issuecomment-509298962
+  delete parserOptions.EXPERIMENTAL_useProjectService;
   delete parserOptions.project;
   delete parserOptions.projects;
```

Comment:
We'll eventually add a new name for the option like `projectService` or something like it. Which would increase this list to four properties.

An alternative to hardcoding names in this plugin would be https://github.com/typescript-eslint/typescript-eslint/issues/8428: importing a shared utility we provide. Is that something you'd find convenient? Either as a part of one of our packages _(easiest for us)_ or as a standalone thing _(we'd have to discuss)_?

Reply:
It's late to add a util for compatibility, what means deleting manually is required for old `@typescript-eslint` versions.

Reply:
For new ones, though, a conditional require would be just fine.

Reply:
sure, sounds good